rackspace/pom.xml

```diff
@@ -41,6 +41,12 @@
       <artifactId>jclouds-sshj</artifactId>
       <version>${jclouds.version}</version>
     </dependency>
+    <dependency>
+       <groupId>org.apache.jclouds.labs</groupId>
+       <artifactId>docker</artifactId>
+       <!-- TODO: Update version to variable jclouds.version on next release -->
```

Comment:
1.9.2 seems broken compared to snapshot. I forgot what the problem was though, let me check